newIDE/app/src/SceneEditor/index.js

```diff
@@ -674,6 +674,11 @@ export default class SceneEditor extends React.Component<Props, State> {
   ) => {
     this.props.initialInstances.moveInstancesToLayer(oldName, newName);
     done(true);
+    this.forceUpdatePropertiesEditor();
```

Comment:
nice! should this be before the done?

Reply:
Nope, this callback is what actually renames the layer after all instances has been updated with the new layer name